data/explorer/beaconMetadata.json

```diff
@@ -1,66 +1,93 @@
 {
-  "0x2c76182b93ab6d486710f745bdbc0acf1c503d78921bd4babd58f93cba1257f7": {
+  "0x09a5873667837598bd0990ba2f53d750d545ce435ecdcd44e0b4c64ab7d7d20d": {
```

Comment:
Yes, excellent catch.
Beacons that have no metadata are supposed to not be rendered in the explorer, but maybe that's too permissive because they could/will start falling through the cracks. 

Maybe adding a script to compare beaconMetadata against active beacons is a good idea. I'll do that on this PR, something small.